index.lit

```diff
@@ -1147,14 +1153,16 @@ void ins(uint16_t instr)
     if (0x0010 & opbit)  // JSR
     {
         uint16_t long_flag = (instr >> 11) & 1;
-        pc_plus_off = reg[R_PC] +  sign_extend(instr & 0x7ff, 11);
         reg[R_R7] = reg[R_PC];
         if (long_flag)
         {
+            pc_plus_off = reg[R_PC] + sign_extend(instr & 0x7FF, 11);
             reg[R_PC] = pc_plus_off;
         }
         else
         {
+            // r1 = (instr >> 6) & 0x7;
+            // Read earlier 
```

Comment:
I don't know that these comments add much value. Isn't this the case for all values for every other instruction?